Source/FikaAmazonAPI/AmazonSpApiSDK/Models/MerchantFulfillment/LabelFormat.cs

```diff
@@ -50,7 +62,13 @@ public enum LabelFormat
         /// Enum ShippingServiceDefault for value: ShippingServiceDefault
         /// </summary>
         [EnumMember(Value = "ShippingServiceDefault")]
-        ShippingServiceDefault = 5
+
+        /// <summary>
+        /// Enum ZPL for value: ZPL
```

Comment:
Please double check this

Reply:
If you make the following request, you receive a response where one of LabelFormat options is ZPL, without adding it as enum option, the deserialization fails:

```
  ShipmentRequestDetails shipmentRequestDetails = new ShipmentRequestDetails()
            {
                AmazonOrderId = "",
                ItemList = new ItemList()
                {
                    new FikaAmazonAPI.AmazonSpApiSDK.Models.MerchantFulfillment.Item()
                    {
                        OrderItemId = "",
                        Quantity = 1
                    }

                },
                ShipFromAddress = new Address()
                {
                    AddressLine1 = "Alcobendas, Madrid",
                    City = "",
                    PostalCode = "28108",
                    Email = "",
                    Phone = "",
                    CountryCode = "ES",
                    Name = ""
                },
                PackageDimensions = new PackageDimensions()
                {
                    Height = 45,
                    Width = 25,
                    Length = 19,
                    Unit = UnitOfLength.Centimeters
                },
                Weight = new Weight()
                {
                    Value = 1400,
                    Unit = UnitOfWeight.G
                },
                ShippingServiceOptions = new ShippingServiceOptions()
                {
                    DeliveryExperience = DeliveryExperienceType.DeliveryConfirmationWithoutSignature,
                    CarrierWillPickUp = false,
                    LabelFormat = LabelFormat.ZPL203
                },
                LabelCustomization=new LabelCustomization()
                {
                    CustomTextForLabel= ""
                }
            };

            var getElegibleShipment = new GetEligibleShipmentServicesRequest()
            {
                ShipmentRequestDetails = shipmentRequestDetails
            };
           var services= amazonConnection.MerchantFulfillment.GetEligibleShipmentServices(getElegibleShipment);
```


```
{
  "payload": {
    "ShippingServiceList": [
                  {
        "ShippingServiceName": "SEUR 24",
        "CarrierName": "SEUR",
        "ShippingServiceId": "SEUR_ES_MFN_24",
        "ShippingServiceOfferId": "5yklm8L6JReB1tXxl52nehD9LalsRiwurLkVNk98ZPMTy3KkXVJJSI3KzXK0u9vSSzoRR8MJ5b8GyMOx93tPpG245DWJR86k8ubf9qjm+zXqw23meb/dEM6Xi/IQSilb/dm4e2jzIr9lKIhOG1Yaw2O7oDF/GJbPlHDBbfdytRnFeEVHM3vIvKBgD7Vzq830dG8uNyiMrHUexb+1PqJEHo972cci1mAqm8uVxXGMzEqp+pI86ocVTi4TCJ9FySRr/w+lhwhWExZpP3nnVRc1oRvFvUk9sHhVLtkAfPlUErc=",
                "ShipDate": "2022-03-09T13:36:25Z",
                "EarliestEstimatedDeliveryDate": "2022-03-10T00:00:00Z",
                        "LatestEstimatedDeliveryDate": "2022-03-10T00:00:00Z",
                "Rate": {
                    "CurrencyCode": "EUR",
          "Amount": 4.53
        },
        "ShippingServiceOptions": {
                              "DeliveryExperience": "DeliveryConfirmationWithoutSignature"
                                                  ,          "CarrierWillPickUp": false
                                                  ,          "LabelFormat": ""
                  }
                        ,"AvailableLabelFormats": [
                    "ZPL203"
          ,                    "ShippingServiceDefault"
          ,                    "ZPL"
          ,                    "PNG"
                            ]
                        ,"AvailableFormatOptionsForLabel": [
                              {
                                                            "LabelFormat": "ZPL203"
                      }
          ,                    {
                                                            "LabelFormat": "ShippingServiceDefault"
                      }
          ,                    {
                                                            "LabelFormat": "ZPL"
                      }
          ,                    {
                                                            "LabelFormat": "PNG"
                      }
                            ]
                      }
      ,            {
        "ShippingServiceName": "Correos Express EPaq24",
        "CarrierName": "CORREOS_EXPRESS",
        "ShippingServiceId": "CORREOS_EXP_MFN_EPAQ24",
        "ShippingServiceOfferId": "IMHT59XPIaVMbOP3KV6G0LAURs4PN7juCtFJXT5opekOzMaVVr32IZW8O1Ewiwh+xyHOUuledfC/2/A8VeWT4Lc+FmEHU/actbI7pD0bEUF8oclvd52pOd1VneWPKkWTWqacfPF+l+E4c4i5uY4jufuezQS5ZC+/fLm1F/AO6RqWHuh/u5CZ4FJ3DSHT68+Lr3aG/io1xfub/6Vi5WcZa865KTDiMu2djytQs3Wfee3TUnX0X8HGsAd+ee6ZxQmQZ8wly1LQV+BgW+dBcBoMeqZrFAXubfXl0HMmtSx8E0u3sEpEFgNn0b40O7rvYGh/ZkUU1VfYsso9vkxosVEs+Q==",
                "ShipDate": "2022-03-09T13:36:25Z",
                "EarliestEstimatedDeliveryDate": "2022-03-10T00:00:00Z",
                        "LatestEstimatedDeliveryDate": "2022-03-10T00:00:00Z",
                "Rate": {
                    "CurrencyCode": "EUR",
          "Amount": 4.62
        },
        "ShippingServiceOptions": {
                              "DeliveryExperience": "DeliveryConfirmationWithoutSignature"
                                                  ,          "CarrierWillPickUp": false
                                                  ,          "LabelFormat": ""
                  }
                        ,"AvailableLabelFormats": [
                    "ZPL203"
          ,                    "ShippingServiceDefault"
          ,                    "PNG"
                            ]
                        ,"AvailableFormatOptionsForLabel": [
                              {
                                                            "LabelFormat": "ZPL203"
                      }
          ,                    {
                                                            "LabelFormat": "ShippingServiceDefault"
                      }
          ,                    {
                                                            "LabelFormat": "PNG"
                      }
                            ]
                      }
                ]
}
```